test/verifiedTests/testSBML/testWriteSBML.m

```diff
@@ -36,15 +45,7 @@
 else
     warning(['The file', fullFileNamePath, ' does not exist and could not be deleted.']);
 end
-
-load('Ec_iJR904.mat')
-model = convertOldStyleModel(model);
```

Comment:
This was not a test for convertOldStyle model but rather something to convert the ecoli model (which is very old). 
But: This test was invalid anyways as it compared an SBML model structure with a COBRA model structure. isSame actually returned false all the time, and it was just that both structs don't have any fields in common that led to numDiff being empty. so essentially, the check for ~any(numDiff) can make sense if one is sure, that both are proper models with potentially different field IDs, but is a bad idea, if that's not sure.